docs/sphinx/source/index.rst

```diff
@@ -6,12 +6,14 @@ pvlib python
 
 pvlib python is a community supported tool that provides a set of
 functions and classes for simulating the performance of photovoltaic
-energy systems. pvlib python was originally ported from the PVLIB MATLAB
-toolbox developed at Sandia National Laboratories and it implements many
-of the models and methods developed at the Labs. More information on
-Sandia Labs PV performance modeling programs can be found at the `PVPMC <https://pvpmc.sandia.gov/>`_. We collaborate with the PVLIB MATLAB project,
-but operate independently of it.
-
+energy systems.  The core mission of pvlib python is to provide open,
+reliable, interoperable, and benchmark implementations of PV system models.
+
+pvlib python started out as a Python translation of the PVLIB MATLAB
+toolbox (developed by the `PVPMC <https://pvpmc.sandia.gov/>`_ at
+Sandia National Laboratories) in 2013 and has grown substantially since then.
+Today it contains code contributions from over a hundred individuals worldwide
+and is maintained by a core group of PV modelers from a variety institutions.
```

Comment:
This is downplaying the continuing major contributions/role of Sandia and DOE quite a bit, it seems.

Reply:
Fair.  Does this strike a better balance?

> pvlib python started out as a Python translation of the PVLIB MATLAB toolbox
(developed by the [PVPMC](https://pvpmc.sandia.gov/) at Sandia National
Laboratories) in 2013.  With continued support from the U.S. Department of
Energy since then, the project has attracted a diverse and growing community
of users and contributors across industry and academia.  
Today it contains code contributions from over a hundred individuals worldwide
and is maintained by a core group of PV modelers from a variety institutions.

Reply:
I would strike "With continued support....DoE" I'm not sure that's factual before 2018

Reply:
DOE supported me as a postdoc 2015-2017. It wasn't the plan, but it made a big difference in pvlib's history.

Reply:
I did not know that. I withdraw my suggestion.

Reply:
I think a separate sentence or two on funding would be better that the "with..." clause. Suggestion to get us started: pvlib has been supported directly and indirectly by DOE funding, companies that allow their employees to contribute, and from personal time.

Reply:
+1 to adding an Acknowledgement of funding